src/data/store/selectors/linkSaturation.selectors.ts

```diff
@@ -4,8 +4,12 @@
 
 import { RootState } from '../createStore';
 
-export const getTotalOpsForGraph = (temporalEpoch: number) => (state: RootState) =>
-    state.linkSaturation.linksPerTemporalEpoch[temporalEpoch]?.totalOps || 0;
+export const getTotalOps = (temporalEpoch: number, chipId?: number) => (state: RootState) =>
+    (chipId !== undefined
+        ? state.linkSaturation.linksPerTemporalEpoch[temporalEpoch]?.chipTotalOps?.[chipId]
```

Comment:
im not sure we shoudl have a scenaio where `chipTotalOps` should not be set. this shoudlnt be store responsibility

Reply:
The null check is only because of typescript and I think the initial render where we don't have the temporal epoch yet.